operations/invoices.md

```diff
@@ -4,11 +4,11 @@
 ## Get invoices
 
 This operation returns a list of invoices.
-Note: This operation needs [Authentication](../essential-guide/authentication.md) and supports following the [JSON:API features](../essential-guide/features.md):
+Note: This operation needs [Authentication](../essential-guide/authentication.md) and supports the following [JSON:API features](../essential-guide/features.md):
 
-- [Relationships](../essential-guide/relationships.md); `invoiceItem`, `user`, `register`, `productVariant`, `invoiceItemModifier` using `include` query parameter.
-- [Filters](../essential-guide/filters.md); `createdAtGt`.
-- [Sparse fieldsets](../essential-guide/sparse-fieldsets.md); supports all fields of `invoices` or related resources with `fields` query parameter.
+- [Relationships](../essential-guide/relationships.md)- `invoiceItems`, `user`, `register`, `productVariants`, `invoiceItemModifiers` using `include` query parameter.
```

Comment:
Are you sure we can include product variants and invoice item modifiers in this call? How would it look like then?

Because for now by the openapi specification we can't. We either need to change that in the specification or this line isn't correct.

Reply:
@MirkoC I just updated the description that we had. I will check the OpenAPI spec and remove it if it's not one of the related resource of invoices. Thanks for pointing it out.